src/Microsoft.DocAsCode.Common/Loggers/LoggerPhaseScope.cs

```diff
@@ -26,14 +21,13 @@ public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
             _originPhaseName = GetPhaseName();
             phaseName = _originPhaseName == null ? phaseName : _originPhaseName + "." + phaseName;
             SetPhaseName(phaseName);
-            if (enablePerformanceScope)
-            {
-                _performanceScope = new PerformanceScope("Scope:" + phaseName, LogLevel.Diagnostic);
-            }
         }
 
+        public LoggerPhaseScope(string phaseName, bool enablePerformanceScope)
+            : this(phaseName, LogLevel.Diagnostic) { }
```

Comment:
Question: This function seems to have never been referenced now, why should we still keep it?